File: challenges/4-fizz-buzz/ruby/peterjpd/desafio-4.rb
valor = ARGV[0].to_i

(1..valor).each do |number|
  if number%3==0 && number%5==0
    puts "FizzBuzz"
  elsif number%3==0
    puts "Fizz"
  elsif number%5==0
    puts "Buzz"
  else
    puts number
  end
end

File: challenges/4-fizz-buzz/php/luisnadachi/desafio4.php
<?php

$limite = (int) $argv[1];

for ($i = 1; $i <= $limite; $i++){
    if (($i % 5 == 0) && ($i % 3 == 0)){
        echo 'FizzBuzz' . PHP_EOL;
    }elseif ($i % 3 == 0){
        echo 'Fizz' . PHP_EOL;
    }elseif ($i % 5 == 0){
        echo 'Buzz' . PHP_EOL;
    }else{
        echo $i . PHP_EOL;
    }
}